.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -27,23 +27,22 @@ Steps to reproduce the behavior:
 A clear and concise description of what you expected to happen.
 
 **Deployment method** *(select with an `X`)*
-- [ ] Docker
+- [ ] Docker Compose
 - [ ] Kubernetes
 - [ ] GoDojo
-- [ ] setup.bash / legacy-setup.bash
 
 **Environment information**
  - Operating System: [e.g. Ubuntu 18.04]
- - DefectDojo Commit Message: [use `git show -s --format="[%ci] %h: %s [%d]"`]
+ - DefectDojo version (see footer) or commit message: [use `git show -s --format="[%ci] %h: %s [%d]"`]
 
-**Sample scan files** (optional)
+**Logs** 
+Use `docker-compose logs` to get the logs and add the relevant sections here showing the error occurring (if applicable).
```

Comment:
Maybe we want to write:

> Use `docker-compose logs` (or similar) to get the logs ...

because not everyone will use docker.

Reply:
thought the same afterwards

Reply:
added